Program.cs

```diff
@@ -64,6 +64,31 @@ public void Begin()
             TypeWriter.WriteLine();
             playerStats();
         }
+
+        private void backGroundTest()
```

Comment:
You should remove this code... it was just there for testing...

did you diff before committing ? ;)